### ISSUE TITLE
viewing_ray, is the width / height the number of pixels?

### ISSUE THREAD

**Issue by DerekCresswell** [karansher/computer-graphics-ray-casting#24]
The camera has a WH for the image plane. So in the viewing ray function are the given parameters for width and height the number of pixels?

**Comment by panuelosj** [comment]
Yes, the const int params are the # of pixels. The camera.width and camera.height are the physical sizes of the image plane in worldspace at distance d from the camera
